lib/manageiq/loggers/container.rb

```diff
@@ -48,6 +48,11 @@ def call(severity, time, progname, msg)
             # :tags => "tags string",
           }.compact
           JSON.generate(payload) << "\n"
+        rescue Encoding::UndefinedConversionError
```

Comment:
I'm not familiar with this exception class.  Is it guaranteed to be required at this point?

Reply:
Yes

```
$ ruby --disable-gems --disable-did_you_mean --disable-rubyopt -e "puts Encoding::UndefinedConversionError"
Encoding::UndefinedConversionError
```